mmv1/templates/terraform/property_documentation.html.markdown.tmpl

```diff
@@ -20,10 +20,13 @@
   {{- end }}
   {{- if or $.WriteOnlyLegacy $.WriteOnly }}
   **Note**: This property is write-only and will not be read from the API.
+
+  * ~> **Note:** One of `{{ slice (underscore $.Name) 0 (sub (len (underscore $.Name)) 3) }}` or `{{ underscore $.Name }}` can only be set.
```

Comment:
It's a little weird to have these two notes right after each other. This isn't necessarily a blocker.